### PR TITLE
Added Link to SunPy

### PR DESCRIPTION
**Pull request by souravsingh**
Added Link to SunPy in the Index file

### CODE REVIEW

**Comment by souravsingh** [comment]
This is the Pull request for the Existing Issue-https://github.com/sunpy/sunpy/issues/1517

File: doc/source/index.rst [hunk 1]
```diff
@@ -15,3 +15,9 @@ free and open-source solar data analysis environment for Python.
  
 .. warning::
     Having issues with links on this page? Make sure there is a trailing slash on the URL *e.g.* `http://docs.sunpy.org/en/latest/`
+    
+
+Link to SunPy
+-----------------
+    
+Wish to stay update with the latest releases? Here is the Link *e.g.* `http://sunpy.org`
```
Comment (dpshelio) [inline doc/source/index.rst]: I would prefer something more accessible from everywhere, like in [astropy](http://astropy.readthedocs.org/), where it's shown in a top bar.

Comment (ehsteve) [inline doc/source/index.rst]: That would require building our own theme which is not on the current horizon.

**Comment by ehsteve** [comment]
Replaced with #1596.

**Pull request closed without merging**